### ISSUE TITLE
Web - Home Page - Poor Color Contrast if Images Do Not Load

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#70]
If the images don't load the user will end up with white text on a white background, which is difficult to read.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. Have you had the images not load? I don't think we would have an issue with images not loading since the images are stored (along with all the other content) on DOL servers. However, we can look into this if there is concern that images will not load.

**Comment by jfc3-dol** [comment]
The problem is if the person internet connection, wifi, etc. is poor at the time the image could not load or if the person turns off images when looking at websites to save bandwidth is when that would be a problem.

There should be an image problem for the Android and iOS applications because I'm assuming they are stored in the application if not they could have the same issue.

**Comment by jfc3-dol** [comment]
In my presentation example, I have a skip navigation link that is not visible until you tab to it which allows the user when clicked to skip over all the other header and navigation links and get them to the main content section of the page. So when they tab next, they go to the next focusable item be that a link, INPUT field, etc.

To see it in action tab until the "Skip to Main Content" link becomes visible at the top of the page. When it's focused, the page will drop down to show the link. Once there click or use the enter key to move you down the page.

http://jfciii.com/presentations/examples/add.html

We can talk about how best to do this with an in-page anchor along with a tabindex="-1" so both the visual and keyboard focus are the same. Some browsers have issues if you don't add the tabindex="-1".

**Comment by trevitalgit** [comment]
added skip to Main Content link
![image](https://user-images.githubusercontent.com/53573580/63014721-2ac06a80-be5d-11e9-84da-53aa8b6ae015.png)

**Comment by jfc3-dol** [comment]
For the attached screenshot simulates if the navigation background images are not sent or received by the user, so they can not see the text. The only reason they show here is that I highlighted them before taking the screenshot. As you can see, there are others that you can't read.

![comply chain - web - if images are not sent or received by the user they can not see the text only reason they show here is because I highlighted them](https://user-images.githubusercontent.com/10423487/64446427-c9c83300-d0a6-11e9-9712-a1f19bb0fc48.png)

**Comment by Abychalil** [comment]
Fixing in progress

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

It looks to be a problem still.

**Comment by jfc3-dol** [comment]
I will close the issue since the color contrast passes for the red and blue background images with white text. Now, if the user turns off images in their browser or they do not load, then you will have white text on a most likely white background, which they will not be able to read.
